### ISSUE TITLE
Cannot re-implement the performance

### ISSUE THREAD

**Issue by JesseYang** [ShuangXieIrene/ssds.pytorch#6]
I have tried to train the ssdlite with Mobilenetv2 as the backbone. As suggested, the pre-trained Mobilenetv2 model is downloaded from another [repo](https://github.com/tonylins/pytorch-mobilenet-v2). Besides changing the weights' name in pre-trained model to load it, I also change the preprocessing of input images. The input image is normalized to to make the mean 0 and standard deviation 1, as an [issue](https://github.com/tonylins/pytorch-mobilenet-v2/issues/9) in the repo of the pre-trained model shows. I didn't make other modifications and followed the Readme to train the ssdlite model for 300 epochs. I used the test.py to test and the mAP after 300 epochs is about 65 to 66. I also tested the pre-trained ssdlite model downloaded from the [73.2 link](https://drive.google.com/open?id=1SBeSIFv5z9AUtwcrJgPI4Xpc8-BCN6ro) in the Readme file, and the mAP is 73.4 when nms threshold is 0.45.

Could you give some suggestions on how to improve the mAP and re-implement the results? Thanks!

**Comment by foreverYoungGitHub** [comment]
The preprocessing will not have a huge influence on the bad result. Actually, normalization always have good result in my experience.

I'm not sure that, when you train the net, do you try to train the extra layers and localization and classification layers first and freeze the base model at the same time?

**Comment by JesseYang** [comment]
Do you mean firstly set the "TRAINABLE_SCOPE" in configuration file as "extras, loc, conf", train some epochs, and then include "base" in the "TRAINABLE_SCOPE"? I have tried that in my tensorflow implementation, but didn't see much difference. In your experience, how much improvement can be achieved by doing this? I will try with this repo. How many epochs should I freeze the base part when training on VOC dataset?

**Comment by foreverYoungGitHub** [comment]
It could get around 2~4% improvement when you freeze the previous layers(sometimes partially basebone). It's hard to say how many epoch it should be freezed. Regularly, I just try 30~50 epochs. 

Besides, I also used the coco as the pretrain model in sometimes, but I'm not sure which model I used to train coco first.

**Comment by bailvwangzi** [comment]
@JesseYang Can you re-implement the mAP 73.x% performance? thanks
